WindowState/sample/build.gradle

```diff
@@ -41,11 +41,10 @@ android {
     }
     composeOptions {
         kotlinCompilerExtensionVersion composeVersion
-        kotlinCompilerVersion '1.5.21'
     }
     packagingOptions {
         resources {
-            excludes += '/META-INF/{AL2.0,LGPL2.1}'
+            excludes += ['META-INF/LGPL2.1', 'META-INF/AL2.0']
         }
     }
 }
```

Comment:
The following dependency(`implementation androidxDependencies.window`)can be removed.

Reply:
Although we don't use it in the WindowState class, I think we still need it for the `rememberWindowState()` [function implementation](https://github.com/microsoft/surface-duo-compose-sdk/blob/main/WindowState/library/src/main/java/com/microsoft/device/dualscreen/windowstate/WindowStateHelper.kt#L25)!